.changeset/happy-cows-moo.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops-backend': patch
+---
+
+Fixed a bug in buildEncodedUrl function where the colon between host name and port was being encoded and breaking the readme card
```

Comment:
```suggestion
Fixed a bug where the azure devops host in URLs on the readme card was being URL encoded, breaking hosts with ports.
```

Function names etc are internal concerns that you don't really know or care about when you read the changelog. The important bit is the user impact.

Reply:
Makes sense, updated. Thanks!